File: app/src/main/java/com/example/quizzlerandroid/Quiz.java
package com.example.quizzlerandroid;

import com.google.gson.annotations.SerializedName;

import org.apache.commons.lang3.StringEscapeUtils;

public class Quiz {
    @SerializedName("question")
    private String question;
    @SerializedName("correct_answer")
    private boolean answer;

    public String getQuestion() {
        return StringEscapeUtils.unescapeHtml4(question);
    }

    public void setQuestion(String question) {
        this.question = question;
    }

    public boolean isAnswer() {
        return answer;
    }

    public void setAnswer(boolean answer) {
        this.answer = answer;
    }
}

File: app/src/main/java/com/example/quizzlerandroid/game/data/Quiz.kt
package com.example.quizzlerandroid.game.data

import com.google.gson.annotations.SerializedName
import org.apache.commons.lang3.StringEscapeUtils

class Quiz {
    @SerializedName("question")
    var question: String = ""
        get() = StringEscapeUtils.unescapeHtml4(field)

    @SerializedName("correct_answer")
    var isAnswer = false
}